src/utils.js

```diff
@@ -492,3 +494,20 @@ export const windowsSpecificFeatures = ['windowsPermissionUsage']
 export function isWindowsSpecificFeature (featureName) {
     return windowsSpecificFeatures.includes(featureName)
 }
+
+export function createCustomEvent (eventName, eventDetail) {
+    // By default, Firefox protects the event detail Object from the page,
+    // leading to "Permission denied to access property" errors.
+    // See https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Sharing_objects_with_page_scripts
+    if (typeof cloneInto === 'function') {
```

Comment:
```suggestion
    if (hasMozProxies) {
```
If we use this, the rollup code removes this code for other browsers.

Reply:
👍  done!